src/main/java/com/google/sps/servlets/GetRankingServlet.java

```diff
@@ -35,7 +36,7 @@
 public class GetRankingServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Set<PlacesSearchResult> hiddenGems = Places.getAllHiddenGems(Places.getAllPlaces());
+    Set<HiddenGem> hiddenGems = Places.getHiddenPlaces(Places.getAllHiddenGems(Places.getAllPlaces()));
```

Comment:
Nit - Maybe change the function name to something like "getPlacesConvertedToHiddenGemsObjects". It is not a big issue, but I found it a bit confusing when reading the sentence. You can even modify the name of my functions to make it clearer, but maybe have something like:
Places.getConvertedPlacesToHiddenGemsObjects(Places.getAllHiddenGems(Places.getAllPlaces()));

Reply:
I changed the function to
Set<HiddenGem> hiddenGems = Places.convertToHiddenGems(Places.getAllHiddenPlaces(Places.getAllPlaces(location)));

Hopefully this is slightly clearer. I feel like getConvertedPlacestoHiddenGemObjects was just a very long name.